### ISSUE TITLE
Recommend tagging of ISBN

### ISSUE THREAD

**Issue by rettinghaus** [TEI-Correspondence-SIG/CMIF#14]
Adding ISBN for (modern) printed editions could help finding them easier with a simple link. 
So adding ISBN to `<bibl>` and tagging it should be recommended: 
`<idno type="ISBN">978-3370001532</idno>`

See #8 

http://www.tei-c.org/release/doc/tei-p5-doc/de/html/ref-idno.html

**Comment by rettinghaus** [comment]
As #8 is resolved, we should rethink what subelements should be allowed within `bibl`. Some are allowed, some not, e.g. `<idno>` is not, what should be changed IMHO.

**Comment by rettinghaus** [comment]
Perhaps even more interesting would be the possibility to add a `<idno type="DOI">` over a `<ref>`.

**Comment by StefanDumont** [comment]
Good Idea, I'm just thinking, that an attribute would be easier to use, e.g. `@sameAs`

**Comment by rettinghaus** [comment]
I would argue that `@sameAs` may be a little too broad, i.e. *take it or leave it*. Whereas `idno` would be more selective, so e.g. correspSearch could process a `DOI` but ignore a `ISBN`.

**Comment by rettinghaus** [comment]
refers to https://github.com/deutschestextarchiv/dtabf/issues/116
